src/pages/contact.md

```diff
@@ -7,12 +7,12 @@ description: Contact Camunda, submit feedback, or find support using the Camunda
 
 There are various channels where you can reach us.
 
-- We encourage everyone to participate in our **community** via the [Camunda Platform community forum](https://forum.camunda.io/), where you can exchange ideas with other Zeebe and Camunda Platform 8 users, as well as the product developers.
+- We encourage everyone to participate in our **community** via the [Camunda community forum](https://forum.camunda.io/), where you can exchange ideas with other Camunda users, as well as the product developers.
 
 - We welcome your **bug** reports and **feature requests** through our community channels mentioned above.
```

Comment:
@nloding is DevRel ok with this line?

Reply:
Made a slight change after getting some more feedback, should be solid now!